arwenmandos/gasSchedules/tomlfiles.go

```diff
@@ -807,32 +775,6 @@ const (
     VerifyBLS              = 5000000
     VerifyEd25519          = 2000000
     VerifySecp256k1        = 2000000
```

Comment:
do not delete these. why did you delete ?

Reply:
they were not necessary. only necessary in gasSchedule v3 I think